docs/manual/java/guide/cluster/PersistentEntity.md

```diff
@@ -69,7 +69,12 @@ The command can be validated before persisting state changes. Use `ctx.invalidCo
 
 @[validate-command](code/docs/home/persistence/Post2.java)
 
-A `PersistentEntity` may also process commands that do not change application state, such as query commands. Such command handlers are registered using `setReadOnlyCommandHandler` of the `BehaviorBuilder`. Replies are sent with the `reply` method of the context that is passed to the command handler function.
+A `PersistentEntity` may also process commands that do not change application state, such as query commands  or commands that are not valid in the entity's currernt state (such as a bid placed after the auction closed). 
+Such command handlers are registered using `setReadOnlyCommandHandler` of the `BehaviorBuilder`. 
+Replies are sent with the `reply` method of the context that is passed to the command handler function.
+
```

Comment:
Hi @TimPigden thanks for expanding this area of the docs!

Just a minor style issue, we try to keep the complete paragraph in a single line or separate paragraphs with a double `CR` .

Here's what the rendered page will look like with the current style.
<img width="680" alt="screen shot 2017-01-31 at 06 12 56" src="https://cloud.githubusercontent.com/assets/762126/22466181/5add437e-e77c-11e6-875a-4a05659afef6.png">

Finally, if you want to run docs locally to get immediate feedback of your changes, you can  `cd $LAGOM_FOLDER/docs; sbt run` in the docs folder (see https://github.com/lagom/lagom/tree/master/docs).